CONTRIBUTING.md

```diff
@@ -116,6 +116,14 @@ Log messages should follow the style:
 
   path/file2:
       Specific changes in file2.
+
+We provide a .gitmessage in the root of the repository.
+Run this command to start using the new commit message template:
+
+```
+git config commit.template /path/to/repo/.gitmessage
```

Comment:
I'd add --local, I checked and it's the default, but it serves as documentation and removes any doubt in case git's command line interface changes.